src/constants.js

```diff
@@ -19,6 +19,7 @@ exports.ACTIONS_PATH = '/actions';
 exports.LOGIN_PATH = '/login';
 exports.LOG_OUT_PATH = '/log-out';
 exports.SETTINGS_PATH = '/settings';
+exports.COMMAND_PATH = '/command';
```

Comment:
I'd suggest calling this `/commands` (plural) for consistency. RESTfully speaking it's a collection of command resources you're adding a command to.